outlines/fsm/guide.py

```diff
@@ -274,3 +307,21 @@ def must_terminate_state(self, state: CFGState) -> bool:
     def copy(self) -> "CFGGuide":
         """Create a copy of the Guide."""
         return CFGGuide(self.cfg_string, self.tokenizer)
+
+
+class BetterFSM:
+    def __init__(self):
+        self.finals = None
+        self.flat_transition_map = None
```

Comment:
Do we still need this?

Reply:
No we shouldn't need this

Reply:
Should I remove everything that's `fsm`-related?